packages/cli/src/util/build/import-builders.ts

```diff
@@ -213,14 +209,21 @@ async function installBuilders(
     ).join(', ')}`
   );
   try {
-    await spawnAsync(
+    const { stderr } = await execa(
       'npm',
       ['install', '@vercel/build-utils', ...buildersToAdd],
       {
         cwd: buildersDir,
         stdio: 'pipe',
+        reject: true,
       }
     );
+    stderr
+      .split('\n')
```

Comment:
Once upon a time I ran into a problem on Windows where the output contained both carriage returns and new lines. Ever since then, I've always split on `/\r?\n/`. I'm not sure if Node.js guards against this these days or if Windows got better. I wish I could recall the exact scenario, but it was probably PowerShell related. In any case, it's probably harmless as is.

Reply:
Added in [00ae98d](https://github.com/vercel/vercel/pull/9289/commits/00ae98d1948869b0f710020316bf0eba2218d5aa)